src/utils.rs

```diff
@@ -157,15 +234,28 @@ impl Console {
 
             if size > 0 {
                 output.write(&buffer[..size]).map_err(|e| {
-                    new_nitro_cli_failure!(
-                        &format!(
-                            "Failed to write data from the console to the given stream: {:?}",
-                            e
+                    let cleanup_result = epoll_cleanup(epoll_fd, self.fd);
+                    match cleanup_result {
```

Comment:
why do you need this match ?

Reply:
because the result of the `epoll_cleanup` function must be consumed

Reply:
why does it need to be consumed ?

Reply:
because in Rust there is a rule: If your function returns a Result<T, E> you must consume it.